http/src/v0/block.rs

```diff
@@ -156,3 +304,84 @@ pub fn stat<T: IpfsTypes>(
         .and(query::<StatQuery>())
         .and_then(stat_query)
 }
+
+use pin_project::pin_project;
```

Comment:
Will use these from the `refs` implementation. At some point these should probably be moved to `support`

Reply:
I think they should be moved right away since they were just copypasted.